### DISCUSSION TITLE
Proposed fixes after the catalog implementation

### DISCUSSION THREAD

**Discussion by matheusps** [vtex/shoreline#462] [Technical RFCs]
- [x] Menu resolver: Lack of support for dividers, and links. Lacks composability and flexibility. The API  is limited and should be reviewed. [More context](https://github.com/vtex/admin-ui/discussions/442).
- [x] Resolvers: When to create resolvers? Are resolvers a good Pattern? 
- [x] Pagination: Review component API.
- [x] Filter: Document how to create custom filters. (improve?)
- [x] Page: Remake!

**Comment by lucasaarcoverde** [comment]
* Table resolvers RFC - https://github.com/vtex/admin-ui/discussions/560
* Page RFC - https://github.com/vtex/admin-ui/discussions/562

**Comment by davicostalf** [comment]
Será que não vale quebrar essa discussão em outras, mesmo que a gente coloque só "WIP" nelas?
Acho que os pontos do Menu e do Filtro poderiam ser issues.

**Comment by matheusps** [comment]
Addressed on Shoreline.
